presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/QueryPreparer.java

```diff
@@ -119,15 +125,24 @@ private static void validateParameters(Statement node, List<Expression> paramete
         }
     }
 
+    private static void checkCondition(boolean condition, ErrorCodeSupplier errorCode, String formatString, Object... args)
```

Comment:
Do you want to move it to a util or something?

Reply:
i inlined this method logic.